data-model/src/main/java/io/micronaut/data/repository/jpa/JpaSpecificationExecutor.java

```diff
@@ -156,6 +156,22 @@
      */
     long count(@Nullable PredicateSpecification<T> spec);
 
+    /**
+     * Returns whether an instance was found for the given {@link QuerySpecification}.
+     *
+     * @param spec The query specification
+     * @return the number of instances.
+     */
+    boolean exists(@Nullable QuerySpecification<T> spec);
```

Comment:
Please add `since` to new methods.

Reply:
New methods should be added to reactive, async, and Koltin coroutines interfaces.